cypher/cypher-docs/src/docs/dev/deprecations-additions-and-compatibility.asciidoc

```diff
@@ -148,6 +273,15 @@ OPTIONS "{" btree-option: btree-value[, ...] "}"
 a|
 Node key and uniqueness constraints with b-tree options are deprecated and need to be migrated before upgrading to 5.0.
```

Comment:
I see that this is not part of this review so it is already in?, If planner is not done we should look over this so we won't scare people into try to use the new types before they are usable

Reply:
yes, there is some updates already in just to make sure we document that it is deprecated at all, https://github.com/neo4j/neo4j-documentation/pull/1256

Reply:
Okay :+1: , will raise this when we get closer to release so we can take a last look and see that the user won't be scared to use their btree indexes